### DISCUSSION TITLE
Adding external text to embed

### DISCUSSION THREAD

**Discussion by yellowcoke** [Rapptz/discord.py#6750] [Q&A] [answered]
Hi everyone,

I'm trying to send an external text with an embed. I mean the external text and embed will be sent in one message object. I did deep search but found nothing.

**Comment by ghost** [comment]
isnt this text means `(edited)`?

**Reply by yellowcoke** [reply]
Yeah but we do not have any work at there.

My purpose is write a text above the embed. But this two should be sent in one message object

**Comment by Luc1412** [comment, marked as answer]
You can just fill the normal content field. Example: await messageable.send("My Text", embed=myembed)

**Reply by yellowcoke** [reply]
Thank you
